### PR TITLE
docs: update documentation for recently implemented features

### PR DESCRIPTION
**Pull request by danshilm**
#### Description
Updated various points in the documentation.
 - Updated mentions of local users
 - Updated mentions of Radarr/Sonarr sync
 - Consistent capitalization of the word 'Sonarr' and 'Radarr'
 - Updated mentions of user roles
 - Updated mentions of 4K requests

#### Screenshot (if UI related)
N/A

#### Todos
None

#### Issues Fixed or Closed by this PR
None

### CODE REVIEW

File: docs/support/faq.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you can't find a solution here, please ask on [Discord](https://discord.gg/Pk
 
 ### I receive 409 or 400 errors when requesting a movie or tv show!
 
-**A:** Verify your are running radarr and sonarr v3. Overseerr was developed for v3 and is not currently backward compatible.
+**A:** Verify your are running Radarr and Sonarr v3. Overseerr was developed for v3 and is not currently backward compatible.
```
Comment (TheCatLady) [inline docs/support/faq.md:11]:
```suggestion
**A:** Verify you are running Radarr and Sonarr v3. Overseerr was developed for v3 and is not currently backward-compatible.
```

File: docs/support/faq.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ The most secure method, but also the most inconvenient, is to set up a VPN tunne
 
 ### Can I make 4K requests?
 
-**A:** 4K requests are not supported just yet but they will be supported in the future!
+**A:** Yes! When adding your 4K Sonarr/Radarr server, tick the `4K Server` checkbox. You also need to tick the `Default Server` checkbox if the Sonarr/Radarr server you're adding is the default one you want your requests to go to for 4K content (there needs to be a default Sonarr/Radarr server for **both** 4K **and** non-4K content).
```
Comment (TheCatLady) [inline docs/support/faq.md:35]:
```suggestion
**A:** Yes! When adding your 4K Sonarr/Radarr server, tick the `4K Server` checkbox. You also need to tick the `Default Server` checkbox if it is the default server you would like to use for 4K content requests. (To enable 4K requests, there need to be default Sonarr/Radarr servers for both 4K content **and** non-4K content.)
```

File: docs/support/faq.md [hunk 3]
```diff
@@ -72,17 +72,17 @@ Perform these steps to verify the media item has a guid Overseerr can match.
 
 ### Can I create local users in Overseerr?
 
-**A:** Not at this time. But it is a planned feature!
+**A:** Head to **Settings -&gt; Users** and hit **Create Local User**. Keep in mind that local user accounts need a valid email address.
```
Comment (TheCatLady) [inline docs/support/faq.md:75]:
```suggestion
**A:** Head to **Users** and hit **Create Local User**. Keep in mind that local user accounts need a valid email address.
```

File: docs/support/faq.md [hunk 4]
```diff
@@ -72,17 +72,17 @@ Perform these steps to verify the media item has a guid Overseerr can match.
...
 
 ### Is is possible to set user roles in Overseerr?
 
-**A:** Unfortunately, this is not possible yet. It is planned!
+**A:** The user roles can be set for each user individually on the **Users** page. The list of permissions is one that is still growing, so if you have any suggestions, [make a feature request](https://github.com/sct/overseerr/issues/new/choose) on GitHub.
```
Comment (TheCatLady) [inline docs/support/faq.md:79]:
```suggestion
**A:** User roles can be set for each user on the **Users** page. The list of assignable permissions is one that is still growing, so if you have any suggestions, [make a feature request](https://github.com/sct/overseerr/issues/new/choose) on GitHub.
```

File: docs/support/faq.md [hunk 5]
```diff
@@ -72,17 +72,17 @@ Perform these steps to verify the media item has a guid Overseerr can match.
...
 
 ## Requests
 
-### I approved a requested movie and radarr didn't search for it!
+### I approved a requested movie and Radarr didn't search for it!
 
-**A:** Check your minimum availability in radarr. If an added item does not meet the minimum availability, no search will be performed. Also verify that radarr did not search for it by checking the radarr logs. Lastly, verify the item was not already being monitored by radarr. Currently there is no state sync with radarr.
+**A:** Check your minimum availability in Radarr. If an added item does not meet the minimum availability, no search will be performed. Also verify that Radarr did not search for it by checking the Radarr logs. Lastly, verify the item was not already being monitored by Radarr.
```
Comment (TheCatLady) [inline docs/support/faq.md:85]:
```suggestion
**A:** Check the minimum availability setting in your Radarr server. If a movie does not meet the minimum availability requirement, no search will be performed. Also verify that Radarr did not perform a search, by checking the Radarr logs. Lastly, verify that the item was not already being monitored by Radarr prior to approving the request.
```

File: docs/support/faq.md (hunk 1 above)
Comment (danshilm) [reply docs/support/faq.md]: Nice catch

**Review by sct — APPROVED** [review]
Awesome. Thanks!

**Pull request merged**